docs/firmware/golioth-firmware-sdk/authentication/certificate-auth.md

```diff
@@ -179,8 +179,12 @@ new device credentials and you are the only one who should have access to it.
 With the public root certificate uploaded to Golioth, and the device
 certificate/key in use in your firmware, your device will automatically connect
 and authenticate to your project. Look in the [Golioth Web
-Console](https://console.golioth.io) to verify a device matching the
-`DEVICE_NAME` you supplied when creating device credentials is now
-present:
+Console](https://console.golioth.io) to verify a device with a name that either matches or is prefixed by the `Certificate_ID` you supplied when creating the device credential:
```

Comment:
@miguel91it can we be consistent with our formatting of `CERTIFICATE_ID` and Certificate ID. I think the former should be use when referring to the value provided in the scripts, and the latter should be used to refer to the attribute of the device. So for example in this case we would use `CERTIFICATE_ID`.
```suggestion
Console](https://console.golioth.io) to verify a device with a name that either matches or is prefixed by the `CERTIFICATE_ID` you supplied when creating the device credential:
```

In the following line
> **immutable** `Certificate ID` field shown in the Summary Page of the device

`Certificate ID` should just be Certificate ID (no backticks).